app/models/sorting/event.js

```diff
@@ -0,0 +1,28 @@
+const getUnixDate = (dateString) => {
```

Comment:
Are the date values not already in UTC string format, and therefore can simply be sorted without conversion?

Reply:
tried to replace, all tests fail when we are just using the string